docs/gcp_alert_policy_labels.md

```diff
@@ -0,0 +1,28 @@
+# GCP Alert Policy Labels
+> ## How Revere understands Google Cloud Monitoring Alert Policies
+
+Revere parses incoming Cloud Monitoring Alert incidents based on the "user labels" associated with the Alert Policy itself.
+
+This avoids using inconsistent naming schemes or the added complexity of attempting to parse alert conditions themselves.
+
+Label keys and values are subject to [Google's format and content restrictions](https://cloud.google.com/resource-manager/docs/creating-managing-labels#requirements).
+
+| Key | Meaning | Example Value |
+|:---:|:-------:|:-------:|
+| `revere-service-name` | "What is the 'short name' of the service?" | `buffer` |
+| `revere-service-environment` | "Where does this instance of the service operate?" | `prod` |
+| `revere-service-degradation` | "What is degraded when this alert policy triggers?" | `uptime` |
```

Comment:
My idea is that while usually the config file will semantically say something like
```
any alert relating to buffer should mean buffer is down
```
I can imagine a lot of scenarios, likely just even in the go-live process for Revere, where we want to say something like
```
alerts matching 'uptime' mean buffer is down, other alerts shouldn't update the public static page
```

I guess in other words I feel like we will end up wanting some mechanism to let Revere differentiate between alert types. All of the labels here are only given meaning by Revere's config file, so Revere itself won't have a notion of uptime versus replica count, it is only if the config file specifically says to handle them differently.

Reply:
I guess another idea would be to have the label here be "severity" or something, and have alerts themselves define the severity rather than leaving it up to Revere's config file. That might be easier to understand from the TF side of things but I don't know if it would make Revere's config file more complicated or less useful

Reply:
> alerts matching 'uptime' mean buffer is down, other alerts shouldn't update the public static page

Should the label be boolean? `revere-processing-enabled` or similar?

Having a free-form text field feels like a recipe for inconsistency and confusion. If we must have it, perhaps it should be called `revere-alert-type`. As someone encountering this label for the first time, I found it confusing, especially with it having a value like `uptime`. I was thinking "Shouldn't all these alerts affect component uptime in StatusPage?"

> I guess in other words I feel like we will end up wanting some mechanism to let Revere differentiate between alert types

I would like to hear more about other use cases for this! Is there a feature in statuspage that can take advantage of this info?

Reply:
Statuspage gives us a few options here, I was worried that boolean fields for these things would get a bit unwieldy. I do like the idea of calling it type; maybe Revere's docs page just defines the possible types.
<img width="229" alt="Screen Shot 2021-09-13 at 4 12 18 PM" src="https://user-images.githubusercontent.com/29168264/133149943-bad42fc9-ffab-4228-b1ce-10a03e65b7f0.png">

Reply:
Just pushed a change here based on this -- https://github.com/broadinstitute/revere/pull/15/files#diff-1c7d4a33e587867144e90be6e3c7958c9ac63ece109946ae8d13efc08a526d05R14

Reply:
Nice! 👍